forecasts.js

```diff
@@ -271,18 +261,18 @@ forecasts.forecastForRegionId = function(regionId, onForecast) {
                         // cache the result
                         forecasts.mostRecentForecasts[regionId] = forecast;
                     }
-                    
+
```

Comment:
thanks for cleaning up all this extra whitespace

Reply:
no worries.